docs/source/Howtos/Beginner-Tutorial/Part1/Beginner-Tutorial-Adding-Commands.md

```diff
@@ -299,9 +299,6 @@ Still in `mygame/commands/mycommands.py`, add a new class, between `CmdEcho` and
 ```{code-block} python
 # in mygame/commands/mycommands.py
 
-:linenos:
-:emphasize-lines: 3,4,11,14,15,17,18,19,21
-
```

Comment:
This should _not_ be removed; it's an intended part of the markdown. The issue is just that this should appear immediately on the line following the `{code-block} python` line (so before the `# ...` comment which is a part of the code example). Possibly the line numbers also need to be updated to match the text since  that top comment was probably added later. 

See [Evennia's documentation-docs about advanced code blocks](https://www.evennia.com/docs/latest/Contributing-Docs.html?highlight=docs#a-more-flexible-code-block) :)